index.html

```diff
@@ -71,5 +71,141 @@ <h1 id="Greeting-text">Hey there. I’m Shedrack!</h1>
     
           </ul>
     </main>
+
+      <div class="icon-x">
+        <div class="arrow-icon">
+        <i class="fa fa-angle-down"></i>
+        </div>
+      </div>
+
+        <div>
+          <p id="text-tag">
+          My Recent Works
+          </p>
+          <hr id="line">
+        </div>
+
+   <div class="wrapper">
+      <div class="content">
+        <section class="projects-container">
+          <div class="project-box">
+              <img class="img-project1" alt="project1-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          <div class="project-box">
+              <img class="img-project2" alt="project2-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          <div class="project-box">
+              <img class="img-project3" alt="project3-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
```

Comment:
Please, modify the list of languages for the projects to match the design of your chosen template.

You'll agree that it looks better. Check the comment above for more on why you need to stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185741283-49f6af4c-5f08-4a7f-9163-a2bfe295cd15.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185741279-c24c6642-0927-48d6-b7df-5b68c0ed926b.png)